getting_started/step_by_step/2_create_player_character.rst

```diff
@@ -204,8 +204,7 @@ specific animations for the character for each direction angle:
    The ``Speak`` animations are optional and only required if your game needs
    them.
 
-For each direction angle, put the name of the matching animation (with the name
-specified in "Adding a walkcycle" above) in the "Animation" field, and choose
+For each direction angle, each slot needs to contain a ``ESCAnimationName`` resource. Put the name of the matching walking animation (with the name specified in "Adding a walkcycle" above) in the "Animation" field, and choose
```

Comment:
```suggestion
Each slot for the above animations needs to contain a ``ESCAnimationName`` resource,
corresponding to each direction angle. Put the name of the matching animations (using
the names specified in "Adding a walkcycle" above) in their respective "Animation" fields,
and choose whether each animation should be played mirrored by selecting the
"Mirrored" checkbox.
```

Reply:
The escanimationname resource should go before anything else, otherwise the user following the tutorial won't understand what to use until they scroll below the images that use the resource.

Reply:
I don't understand what your argument is: The suggested change I made follows your change, but uses slightly different verbiage. ESCAnimationName still appears in exactly the same place you had it.

Reply:
Sure thing, I'm new to pull requests so I wasn't entirely sure what was approved or not, sorry!

Reply:
Ah, no worries. Yeah, GitHub's PR interface takes a little getting used to. So far, nothing's been "approved", but I have requested a change to be made (above) before I decide on whether to accept the PR.

Note that, for this project, approval requires "acceptance" from at least two of the devs.

Reply:
Ahh thanks for the explanation!

Reply:
I'd actually like to overhaul this part of the documentation as I think we could explain the world angle to ESCAnimation resource to actual animation mechanism a lot better. Happy to run with this in the interim.